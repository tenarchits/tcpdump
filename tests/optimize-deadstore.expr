((ip[(((ip[0]&0xf)<<2) + ((ip[((ip[0]&0xf)<<2)+1] & 0x80) >> 5) + 12)+4 +9]=17) and (ip[(((((ip[0]&0xf)<<2) + ((ip[((ip[0]&0xf)<<2)+1] & 0x80) >> 5) + 12)+4 +((ip[(((ip[0]&0xf)<<2) + ((ip[((ip[0]&0xf)<<2)+1] & 0x80) >> 5) + 12)+4]&0xf)<<2))+20):4]=0x7fffffff))
