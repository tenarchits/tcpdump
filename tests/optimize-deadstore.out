    1  20:09:01.743430 IP 78.140.0.12 > 217.29.94.29: GREv1, call 49152, seq 2983, ack 3040, length 66: IP 109.123.190.30.52583 > 78.140.3.124.666: UDP, length 18
